systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -542,11 +552,11 @@ def configure_ipsec(self, obj):
             secret.commit()
             file.commit()
             logging.info("Configured vpn %s %s", leftpeer, rightpeer)
-            CsHelper.execute("ipsec auto --rereadall")
-            CsHelper.execute("ipsec auto --add vpn-%s" % rightpeer)
+            CsHelper.execute("ipsec reload")
+            #CsHelper.execute("ipsec --add vpn-%s" % rightpeer)
             if not obj['passive']:
-                CsHelper.execute("ipsec auto --up vpn-%s" % rightpeer)
-        os.chmod(vpnsecretsfile, 0o400)
+                CsHelper.execute("sudo nohup ipsec up vpn-%s &" % rightpeer)
```

Comment:
@rhtyd If we start/stop the existing tunnels get effected. Usually  ipsec up command takes long time, if do not  use the nohup the script will  hang for long time it may result timeout for resource command.